ta4j-core/src/main/java/org/ta4j/core/indicators/MACDIndicator.java

```diff
@@ -36,6 +36,8 @@
  */
 public class MACDIndicator extends CachedIndicator<Num> {
 
+    private static final long serialVersionUID = -6899062131135971403L;
```

Comment:
remove this line. we don't use serialVersionUID.

Reply:
True, fixed